sbt-gcs-plugin/src/main/scala/org/latestbit/sbt/gcs/GcsPlugin.scala

```diff
@@ -63,18 +65,30 @@ object GcsPlugin extends AutoPlugin {
       super.projectSettings
 
   private def loadGoogleCredentials(
-      gcsCredentialsFilePath: Option[Path]
+      gcsCredentialsFilePath: Option[Path],
+      gcsCredentialAccessToken: Option[String] = None
   )( implicit logger: Logger, projectRef: ProjectRef ): GoogleCredentials = {
     val scopes: java.util.Collection[String] =
       ImmutableList.copyOf( GoogleCredentialsScopes.asJavaCollection.iterator() )
-    gcsCredentialsFilePath
-      .orElse( lookupGoogleCredentialsInSbtDir() )
-      .map { path =>
-        logger.debug( s"Loading Google credentials from: ${path.toAbsolutePath.toString} for ${projectRef.toString}" )
-        GoogleCredentials
-          .fromStream( new FileInputStream( path.toFile ) )
-          .createScoped( scopes )
-      }
+    val optGoogleCred =
+      if (gcsCredentialAccessToken.isDefined)
```

Comment:
Nit: this is written in non-idiomatic Scala style. In Scala `.orElse {}` on Option is used for this instead of `isDefined`. You can leave it, so I can change it myself - not a big deal.